spec/support/fixtures.rb

```diff
@@ -114,3 +118,24 @@ class CustomParserModel < Base
     attribute  :age, parser: FixNumParser
   end
 end
+
+module Fixtures
+  class Association < PDC::Base
+    # stub the connection
+    SITE = 'https://pdc.host.dev.eng.pek2.redhat.com/'.freeze
+    self.connection = Faraday.new(url: SITE) do |faraday|
+      faraday.request   :json
+      faraday.use       PDC::Response::Parser
+      faraday.adapter   Faraday.default_adapter
+    end
+  end
+
+  class ProductVersion < Association
+    has_many :releases, uri: 'rest_api/v1/releases/?product_version=:product_version_id'
```

Comment:
Isn't the URI given by the name spacing of the association? E.g. 
```Ruby
pv = PDC::V1::Release::ProductVersion.find('foo')
```
which would translate the URL the above. So not really sure why an additional URI needs to be configured...

Reply:
The URI is constructed by several parts, the configured site, the uri and the query parameter. The uri has default value, but it is not proper for current PDC server, we need to customize it. So that is why additional uri needed